### DISCUSSION TITLE
[FEATURE REQ] Update Commands

### DISCUSSION THREAD

**Discussion by ostermayer** [minibolt-guide/minibolt#6] [Ideas]
### What & why

Many Users will also need help with update commands for each of the installed applications 

the addition of wget and tar commands for install of new version and removal of old version will be helpful and educational for many users

**Comment by regiregi22** [comment]
I have already suggested Twofaktor creating a dedicated section for updating Minibolt, instead of indicating how to update each service in it's own section (and in its current state, it is not even very straightforward for a novice person). 
When you think about updating your node, you think of it as a whole system to update, instead of planning updating each service as a separate task.

I use this guide I made myself for a similar deployment on a RaspberryPi instead of an x64 system. It could be easily adapted and rearranged, as the OS is the same Debian based Ubuntu:
https://github.com/regiregi22/Raspibolt-things/blob/main/Raspibolt%20Upgrade

**Comment by twofaktor** [comment]
Hello guys!!

Thank you for your interest in the MiniBolt project and for contributing to the improvements

This is an amazing suggestion. I think I thought about it once. Directly to the roadmap, thanks @regiregi22 for the reference and @ostermayer for the suggestion! MiniBolt to the moon!! :rocket:

**Comment by twofaktor** [comment]
Already in the [roadmap](https://github.com/orgs/minibolt-guide/projects/1?pane=issue&itemId=41917837), status "IN PROGRESS"
